apps/site/src/views/Home/sections/Landing/Landing.tsx

```diff
@@ -1,13 +1,14 @@
 "use client";
 
-import ApplyButton from "../../components/ApplyButton/ApplyButton";
 
+import ApplyButton from "@/views/Home/components/ApplyButton/ApplyButton";
+import Title from "@/views/Home/components/Title/Title";
 import styles from "./Landing.module.scss";
 
 function Landing() {
 	return (
 		<div className={styles.landing}>
-			<h1>ZotHacks 2023</h1>
```

Comment:
Issue: By removing the heading, we lose an accessible page structure. We could keep the heading but make it [visually hidden](https://getbootstrap.com/docs/5.3/helpers/visually-hidden/).